gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -594,8 +594,9 @@ message EntitySelector {
   optional string route_id = 2;
   // corresponds to route_type in GTFS.
   optional int32 route_type = 3;
-  optional TripDescriptor trip = 4;
```

Comment:
`trip` needs to be left in the  .proto to show that number `4` can't be used anymore, but `[deprecated=true]` needs to be added.  See https://github.com/google/transit/issues/109#issuecomment-442476116 for discussion.